dailymotion-player-objc/DMPlayerViewController.m

```diff
@@ -282,6 +298,23 @@ - (void)api:(NSString *)method {
     [self api:method arg:nil];
 }
 
+- (NSString *) getAPIReadyWarnMessageForMethod:(NSString *)method {
+
+    NSString * param = [@{
+      @"play"         : @"autoplay",
+      @"toggle-play"  : @"autoplay",
+      @"seek"         : @"start",
+      @"quality"      : @"quality",
+      @"muted"        : @"muted",
+      @"toggle-muted" : @"muted"
+    } valueForKey:method];
+
+    return (param != nil) ? [NSString stringWithFormat:@"Warning [DMPlayerViewController]: \n"
```

Comment:
Same here for the explicit `!= nil`, can just check for `return param ? … : nil`